Source/Core/Jobs/EventPostsJob.cs

```diff
@@ -89,9 +89,10 @@ public class EventPostsJob : QueueJobBase<EventPost> {
 
                 // Discard any events over there limit.
                 events = events.Take(eventsToProcess).ToList();
-
-                // Increment by count - 1 since we already incremented it by 1 in the OverageHandler.
-                await _organizationRepository.IncrementUsageAsync(project.OrganizationId, false, events.Count - 1, applyHourlyLimit: false).AnyContext();
+                
+                // Increment the count if greater than 1, since we already incremented it by 1 in the OverageHandler.
+                if (events.Count > 1)
+                    await _organizationRepository.IncrementUsageAsync(project.OrganizationId, false, events.Count - 1, applyHourlyLimit: false).AnyContext();
```

Comment:
Yeah, but that's not what we are currently doing. When we call the increment overage we just see a boolean if anything was blocked, but not how many.